bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/styles.js

```diff
@@ -18,7 +19,6 @@ const Content = styled.div`
     right: 0;
     bottom: 0;
     left: 0;
-    border: 5px solid ${colorPrimary};
```

Comment:
This line is responsible for displaying the talking indicator border when the video is loaded.

You can make the new styles work when the video is loaded by changing it to `border-radius: 10px;border: 2px solid ${colorPrimary};` instead of removing it completely.

Reply:
Whoops... Fixed in https://github.com/bigbluebutton/bigbluebutton/pull/14938/commits/1d51a6b47cb021ae360961dadef29e52e495da87